djmoney/contrib/django_rest_framework/fields.py

```diff
@@ -40,6 +47,12 @@ def validate(self, value):
             amount = value.amount if isinstance(value, Money) else value
             return super(MoneyField, self).validate(amount)
 
+        def field_from_native(self, data, files, field_name, into):
+            super(MoneyField, self).field_from_native(data, files, field_name, into)
+            currency = data.get('{}_currency'.format(field_name), None)
```

Comment:
Hi @Stranger6667 , why are you still supporting Python 2.6 (11 yo, not supported by Django 1.8.18/LTS version) or DRF < 3 (any commit in the last 2 years)?